src/Shared/Throw/Throw.cs

```diff
@@ -943,7 +943,7 @@ public static double IfGreaterThanOrEqual(double argument, double max, [CallerAr
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     public static double IfOutOfRange(double argument, double min, double max, [CallerArgumentExpression(nameof(argument))] string paramName = "")
     {
-        if (argument < min || argument > max)
+        if (!(min <= argument && argument <= max))
```

Comment:
I suggest adding a comment providing the background to avoid someone inverting the check later.
Also add a test? :)

Reply:
Done.